action.sh

```diff
@@ -12,13 +12,16 @@ export PATH="${PATH}:/semantic-release/.venv/bin"
 # Change to configured directory
 cd "${INPUT_DIRECTORY}"
 
+export GIT_COMMITER_NAME="${GIT_COMMITER_NAME:="github-actions"}"
```

Comment:
```suggestion
export GIT_COMMITER_NAME="${INPUT_GIT_COMMITER_NAME:="github-actions"}"
```

Reply:
good catch, i merged in kevmo's changes wrong :oof: